tiledb/sm/query/deletes_and_updates/serialization.cc

```diff
@@ -155,4 +155,75 @@ QueryCondition deserialize_condition(
       deserialize_condition_impl(deserializer));
 }
 
+void serialize_update_values_impl(
+    const std::vector<UpdateValue>& update_values, Serializer& serializer) {
+  serializer.write<uint64_t>(update_values.size());
+  for (const auto& update_value : update_values) {
+    uint64_t field_name_size = update_value.field_name().length();
+    serializer.write<uint64_t>(field_name_size);
+    serializer.write(update_value.field_name().data(), field_name_size);
+
+    uint64_t value_size = update_value.view().size();
+    serializer.write<uint64_t>(value_size);
+    serializer.write(update_value.view().content(), value_size);
+  }
+}
+
+storage_size_t get_serialized_update_condition_and_values_size(
+    const tdb_unique_ptr<tiledb::sm::ASTNode>& node,
+    const std::vector<UpdateValue>& update_values) {
+  SizeComputationSerializer size_computation_serializer;
+  serialize_condition_impl(node, size_computation_serializer);
+  serialize_update_values_impl(update_values, size_computation_serializer);
+
+  return size_computation_serializer.size();
+}
+
+std::vector<uint8_t> serialize_update_condition_and_values(
```

Comment:
You plan to use those serialize/deserialize in a future PR?

Reply:
Yes, the update strategy will use those functions directly.